Base/res/js/Spreadsheet/runtime.js

```diff
@@ -1,8 +1,79 @@
+// FIXME: Figure out a way to document non-function entities too.
+class Position {
+    constructor(column, row, sheet) {
+        this.column = column;
+        this.row = row;
+        this.sheet = sheet ?? thisSheet;
+        this.name = column + "" + row;
+    }
+
+    static from_name(name) {
+        let sheet = thisSheet;
+        let obj = sheet.parse_cell_name(name);
+        return new Position(obj.column, obj.row, sheet);
+    }
```

Comment:
Honestly blows my mind that LibJS supports all of this natively!

Reply:
Very true!
I was honestly surprised when I saw nothing blowing up after adding this class.